src/Components/Conversation.tsx

```diff
@@ -248,8 +327,89 @@ export const Conversation: React.FC<IProps> = ({
                   }}
                   source={m.message.content}
                 />
+
+                {displayActions && m.message.role === "assistant" && (
+                  <div className={styles["actions-container"]}>
+                    {Object.values(EnumFiabilityMessage).map(
+                      (fiability_key) => (
+                        <div
+                          onClick={() => {
+                            const send =
+                              m.fiability === fiability_key
+                                ? "DEFAULT"
+                                : fiability_key;
+
+                            handleFiabilityMessage(m, send);
+                          }}
+                          className={styles["btn-container-fiability"]}
+                        >
+                          {fiability_key === "BAD" ? (
```

Comment:
Why not use, const for that ?
```suggestion
                          {fiability_key === FiabilityEnum.BAD ? (
```